### ISSUE TITLE
ste.button possible? (not for download)

### ISSUE THREAD

**Issue by AliciaGro** [PaleNeutron/streamlit-ext#4]
Hello again, and thanks for all your work! 
I would like to ask you if it's possible to use a normal button with streamlit-ext, I would like to use it to copy a dataframe to clipboard, but everytime I do it with a normal st.button everythin realoads :(

something like this:

```
if ste.button('Copy to clipboard'):
            df.to_clipboard(excel=True, index=False)
```

Could it be possible?

**Comment by PaleNeutron** [comment]
Yes, I think it is possible. But it will cause some side effect beyond your use case so I don not implement it.

If a button can remember it's states, it will work like a check box indeed, maybe worse, you can not 'uncheck' it.
